package.json

```diff
@@ -2348,7 +2347,6 @@
         "ts-node": "^8.3.0",
         "tsconfig-paths-webpack-plugin": "^3.2.0",
         "tslib": "^2.3.0",
-        "type-check": "^0.4.0",
```

Comment:
Removing this fixes errors thrown by `npm list --production --parseable --depth=99999 --loglevel=error` which is run by `vsce` when building the vsix package